### ISSUE TITLE
Lombok plugin "generateLombokConfig.enabled = false" is ignored

### ISSUE THREAD

**Issue by sanyarnd** [kelloggm/checkerframework-gradle-plugin#46]
I have the following build file:
```kotlin
...
subprojects {
    apply(rootProject.file("gradle/lombok.gradle"))
    apply(rootProject.file("gradle/checker-framework.gradle"))
...
}
...
```
lombok.gradle:
```groovy
apply plugin: 'io.freefair.lombok'
generateLombokConfig.enabled = false

lombok {
    version = '1.18.8'
}
```
checker-framework.gradle:
```groovy
apply plugin: 'org.checkerframework'

checkerFramework {
    excludeTests = true
    checkers = [
            'org.checkerframework.checker.nullness.NullnessChecker',
            'org.checkerframework.checker.units.UnitsChecker'
    ]
}
```
For some reason, when I enable `checker-framework.gradle` the lombok's `generateLombokConfig.enabled = false` is ignored and I have bunch of lombok.config generated files all over my modules.

**Comment by sanyarnd** [comment]
Sorry for false report, the issue is Kotlin-DSL, seems like it has some poor interop or something. Pure groovy script works like charm.

**Comment by sanyarnd** [comment]
Apparently I closed the issue prematurely, I cannot figure why, but the invalid behavior is still present.

**Comment by kelloggm** [comment]
The line you pointed out does, in fact, generate a `lombok.config`. Our plugin does that intentionally, in an effort to ensure that `@Generated` annotations are placed in delombok'd source code, so that a checker can differentiate between code generated by lombok and code written by the user.

You're correct that we should respect an existing `lombok.config` that doesn't permit generation. I'll see if I can fix it.
